### PR TITLE
Minor edits to top-level docs

### PR DESCRIPTION
**Pull request by RyanFrantz**
- Includes typo and grammar edits for all the top-level docs currently found under the **Docs** header.
- Adds or rearranges some of the context for topics to clarify the scope and direction of the tutorials/examples.

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ The `type Query { }` block declares the type, `hello` is the field definition, `
 
 ## Validating schema
 
-Ariadne provides tiny `gql` utility function that takes single argument: GraphQL string, validates it and raises descriptive `GraphQLSyntaxError`, or returns the original unmodified string if its correct:
+Ariadne provides the `gql` utility function to validate schema. It that takes a
+single argument: a GraphQL string, like the following example.
```
Comment (rafalp) [inline docs/intro.md]: Is the linebreak in the middle of sentence purposeful here?

File: docs/intro.md [hunk 2]
```diff
@@ -80,9 +85,22 @@ The above code is perfectly valid, with a minimal resolver meeting the requireme
 
 Real-world resolvers are rarely that simple: they usually read data from some source such as a database, process inputs, or resolve value in the context of a parent object. How should our basic resolver look to resolve a client's user agent?
 
-In Ariadne every field resolver is called with at least two arguments: `obj` parent object, and the query's execution `info` that usually contains the `context` attribute that is GraphQL's way of passing additional information from the application to its query resolvers.
+In Ariadne every field resolver is called with at least two arguments: the
+query's parent object, and the query's execution `info` that usually contains
```
Comment (rafalp) [inline docs/intro.md]: Another odd linebreak.

File: docs/intro.md [hunk 3]
```diff
@@ -80,9 +85,22 @@ The above code is perfectly valid, with a minimal resolver meeting the requireme
...
+a `context` attribute. The `context` is GraphQL's way of passing additional
+information from the application to its query resolvers.
```
Comment (rafalp) [inline docs/intro.md]: Please remove explicit linebreaks from this paragraph.

File: docs/intro.md [hunk 4]
```diff
@@ -80,9 +85,22 @@ The above code is perfectly valid, with a minimal resolver meeting the requireme
...
+
+> In the above example, note the `*_` argument in the resolver's method signature.
+> The underscore is a convention used in many languages (including Python) to
+> indicate a variable that will not be used. The asterisk prefix is Python syntax
+> that informs the method it should expect a variable-length argument list. In
+> effect, the above example is throwing away any arguments passed to the resolver.
+> We've used that here, to simplify the example so that you can focus on its
+> purpose.
```
Comment (rafalp) [inline docs/intro.md]: Please remove explicit linebreaks from this quote block.

We could also bolden first sentence in this block using strong emphasis syntax (`**text**`) and then put blank line between it and rest of the block - this produces fancy "titled yellow box" effect without introducing headers into hierarchy.

File: docs/intro.md [hunk 5]
```diff
@@ -127,9 +145,15 @@ You pass it your type definitions and resolvers that you want to use:
...
+In Ariadne the process of adding the Python logic to GraphQL schema is called
+*binding to schema*, and special types that can be passed to
+`make_executable_schema`'s second argument are called *bindables*. `QueryType`
+(introduced earlier) is one of many *bindables* provided by Ariadne that
+developers will use when creating their GraphQL APIs.
 
-In our first API we are passing only single instance to the `make_executable_schema`, but most of your future APIs will likely pass list of bindables instead, for example:
+In our first API we passed only a single bindable to the
+`make_executable_schema`, but most of your future APIs will likely pass a list
+of bindables instead, for example:
```
Comment (rafalp) [inline docs/intro.md]: Even more explicit linebreaks ;)

File: docs/mutations.md [hunk 6]
```diff
@@ -4,11 +4,12 @@ title: Mutations
 ---
 
 
-So far all examples in this documentation have dealt with `Query` type and reading the data. What about creating, updating or deleting?
+All the previous examples in this documentation have dealt with the `Query` root
+type and reading data. What about creating, updating or deleting data?
```
Comment (rafalp) [inline docs/mutations.md]: Another odd linebreak.

File: docs/mutations.md [hunk 7]
```diff
@@ -84,7 +87,11 @@ def resolve_logout(_, info):
 
 ## Mutation payloads
 
-`login` and `logout` mutations introduced earlier in this guide work, but give very limited feedback to the client: they return either `false` or `true`. The application could use additional information like an error message that could be displayed in the interface after mutation fails, or an updated user state after a mutation completes.
+The `login` and `logout` mutations introduced earlier in this guide work, but
+give very limited feedback to the client: they return either `False` or `True`.
+The application could use additional information like an error message that
+could be displayed in the interface if the mutation request fails, or a
+user state updated after a mutation completed.
```
Comment (rafalp) [inline docs/mutations.md]: The whole block has explicit line breaks.

File: docs/intro.md (hunk 1 above)
Comment (RyanFrantz) [reply docs/intro.md]: It's markdown which tends to follow a convention of 80 characters per written line, for legibility in your favorite editor. When it renders in a page/browser, however, it looks continuous. Rendered line breaks require a blank line in the original content.

Comment (rafalp) [reply docs/intro.md]: I understand that, but we are not following this convention on Ariadne docs. Its much easier to edit when its single long paragraph wrapped by editor with single `>` sign in front of it.

File: docs/intro.md [hunk 8]
```diff
@@ -48,7 +48,10 @@ type_defs = gql("""
 """)
 ```
 
-If we try to run the above code now, we will get an error pointing to our incorrect syntax within our `type_defs` declaration:
+`gql` validates the schema and raises a descriptive `GraphQLSyntaxError`, if there is an issue, or returns the original unmodified string if it is correct.
+
+
```
Comment (rafalp) [inline docs/intro.md]:
```suggestion
```

**Comment by rafalp** [comment]
Excellent work, thank you!

**Pull request merged**